Source/WTF/wtf/RunLoop.h

```diff
@@ -129,7 +128,7 @@ class WTF_CAPABILITY("is current") RunLoop final : public SerialFunctionDispatch
     static void registerRunLoopMessageWindowClass();
 #endif
 
-    class TimerBase {
+    class TimerBase : public ThreadSafeRefCounted<TimerBase> {
         WTF_MAKE_FAST_ALLOCATED;
```

Comment:
Remove this and add this to the each final TimerBase classes (not for DispatchTimer since ThreadSafeRefCounted<> has one).